### DISCUSSION TITLE
Getting this error when first time setting up dashlane cli

### DISCUSSION THREAD

**Discussion by rishabhrshanbhag** [Dashlane/dashlane-cli#272] [Q&A] [answered]
`error: table device has 10 columns but 11 values were supplied` 

dcli -V : 6.2424.1
Machine : Apple m2 max

**Comment by Mikescops** [comment, marked as answer]
Hello @rishabhrshanbhag,

Can you logout (say yes to clear the local data) and login again?

Thanks!

**Reply by rishabhrshanbhag** [reply]
Yes. It worked! Thanks @Mikescops

**Comment by IPCli** [comment]
After importing the data, I found that there were errors in the import process. When I wanted to delete all the data, there was a very convenient way, but I had to choose one by one, which was a bit troublesome. Is there any other way to reset the account? Thank you

**Reply by Mikescops** [reply]
Hello. you can use this link to reset your account: https://app.dashlane.com/#/reset-account

If you want to read more before doing so, here is the help center page about it: https://support.dashlane.com/hc/en-us/articles/115003111325-I-forgot-my-Master-Password#reset_account_main
